shared/ios/Keybase/MediaUtils.m

```diff
@@ -146,54 +129,37 @@ + (void) processVideoFromOriginal:(NSURL*)url completion:(ProcessMediaCompletion
   CGImageRef cgOriginal = [generateImg copyCGImageAtTime:time actualTime:NULL error:&error];
   if (error != nil) {
     CFRelease(cgOriginal);
-    completion(error, nil, nil);
-    return;
-  }
-  
-  // TODO: this is just the original frame. Figure out how to make thumbnail work here.
-  /*
-  CGImageSourceRef cfSouruce = CGImageSourceCreateWithDataProvider(CGImageGetDataProvider(cgOriginal), nil);
-  error = [MediaUtils _scaleDownCGImageSourceRef:cfSouruce dstURL:thumbnailURL options:[MediaUtils _thumbnailImageOptions]];
-  CGImageRelease(cgOriginal);
-  if (error != nil) {
-    NSLog(@"scale down error: %@", error);
-    completion(error, nil, nil);
-    return;
-  }
-   */
-  NSData * thumbnail = UIImageJPEGRepresentation([UIImage imageWithCGImage:cgOriginal], 0.85);
-  CFRelease(cgOriginal);
-  BOOL OK = [thumbnail writeToURL:thumbnailURL atomically:true];
-  if (!OK) {
-    completion([NSError errorWithDomain:@"MediaUtils" code:1 userInfo:@{NSLocalizedDescriptionKey:@"error getting thumbnail for video"}], nil, nil);
+    completion(error, nil);
     return;
   }
   
   BOOL needScaleDown = [MediaUtils _needScaleDownAsset:asset];
   if (!needScaleDown) {
     [MediaUtils _stripVideoExifAndCompleteAsset:asset originalURL:url completion:^(NSError * _Nullable error) {
       if (error != nil) {
-        completion(error, nil, nil);
```

Comment:
here and below on error would fall through and call the other completion handler accidentally